package.json

```diff
@@ -184,6 +186,16 @@
         "command": "azure-iot-toolkit.updateModuleTwin",
         "title": "Update Module Twin",
         "category": "Azure IoT Edge"
+      },
+      {
+        "command": "azure-iot-toolkit.generateSasTokenForDevice",
+        "title": "Generate SAS Token for Device",
```

Comment:
It may also be Module later? So how about Generate SAS Token?

Reply:
Currently, we have 'Generate SAS Token for IoT Hub' and 'Generate SAS Token for Device'. And the implementation is a little different. If we have SAS Token for Module, it would mostly be a separate command.